### PR TITLE
Improve Devtools Docs based on installation in Docker

### PR DESCRIPTION
**Pull request by dlaird-ovo**
## Updated DevTools Plugin Docs

This is a few changes to DevTools documentation based on my usage with Backstage in a docker container

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by dlaird-ovo** [comment]
@ahhhndre - this is a PR for DevTools Docs - hope this is helpful.

**Comment by dlaird-ovo** [comment]
Do you need a changeset for Doc changes? I read this `Changesets are also not needed for changes that do not affect the published version of each package, for example changes to tests or in-line source code comments.` in CONTRIBUTING.md and assumed it was not required.

File: plugins/devtools/README.md [hunk 1]
```diff
@@ -20,6 +20,20 @@ Lists the configuration being used by your current running Backstage instance.
 
 ![Example of Config tab](./docs/devtools-config-tab.png)
 
+#### Backstage Version Reporting
```
Comment (awanlin) [inline plugins/devtools/README.md:23]: The content and formatting of this is great, I just think it should be moved up to be under the `Info` heading as that's where the Backstage version is shown.

Comment (awanlin) [reply plugins/devtools/README.md:23]: Just below the image

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this contribution @dlaird-ovo, this is super helpful! 🚀 

Overall very happy with this just one nit about where one section is

**Comment by awanlin** [comment]
Regarding the changeset: if you can add it that would be good, it just makes sure that a new package gets created which then updates things on NPM. People do use the `README` content that shows up there so good to keep it up to date

**Review by awanlin — APPROVED** [review]
Awesome! Thanks again @dlaird-ovo 🚀

**Comment by awanlin** [comment]
Hi @benjdlambert, these are just changes to the `README`. I'm happy with them, if you could give them a quick look that would be great. Thanks in advance!

**Comment by awanlin** [comment]
Not stale as it's ready to go, just needs to be approved and merged. Can you take a look @benjdlambert, please?

**Review by benjdlambert — APPROVED** [review]
Apologies for the delay! I've been off since last week, just catching up! :pray:

**Comment by awanlin** [comment]
Thanks again @dlaird-ovo and @benjdlambert 🚀

**Pull request merged**